src/Microsoft.AspNet.Authentication.OAuth/OAuthAuthenticationOptions.cs

```diff
@@ -77,7 +69,7 @@ public string Caption
         /// <value>
         /// The back channel timeout.
         /// </value>
-        public TimeSpan BackchannelTimeout { get; set; }
+        public TimeSpan BackchannelTimeout { get; set; } = TimeSpan.FromSeconds(60);
```

Comment:
Will FxCop even complain if we have no "explicit" constructor? ('cuz everything is set using the new C# 6 syntax)